libraries/ts-command-line/src/CommandLineDefinition.ts

```diff
@@ -21,9 +21,20 @@ export interface IBaseCommandLineDefinition {
    * Documentation for the flag, that will be shown when invoking the tool with "--help"
    */
   description: string;
+
+  /**
+   * If true and no "getDefaultValue" function is provided, the parameter must be provided.
+   */
+  required?: boolean;
+
+  /**
+   * A default value for the parameter if no value was provided. If this property is undefined and the "required" flag
```

Comment:
>A default value for the parameter if no value was provided. [](start = 5, length = 59)

Why should this be modeled by ts-command-line?  Can't the application simply expose this logic?

Does the command-line help somehow indicate the default value?